server/src/internalClusterTest/java/org/opensearch/cluster/SpecificClusterManagerNodesIT.java

```diff
@@ -255,19 +256,24 @@ public void testElectOnlyBetweenClusterManagerNodes() throws Exception {
         client().execute(AddVotingConfigExclusionsAction.INSTANCE, new AddVotingConfigExclusionsRequest(clusterManagerNodeName)).get();
         // removing the cluster-manager from the voting configuration immediately triggers the cluster-manager to step down
         assertBusy(() -> {
-            assertThat(
-                internalCluster().nonClusterManagerClient()
-                    .admin()
-                    .cluster()
-                    .prepareState()
-                    .execute()
-                    .actionGet()
-                    .getState()
-                    .nodes()
-                    .getClusterManagerNode()
-                    .getName(),
-                equalTo(nextClusterManagerEligableNodeName)
-            );
+            Supplier<String> clusterManagerName = () -> {
+                try {
+                    return internalCluster().nonClusterManagerClient()
+                        .admin()
+                        .cluster()
+                        .prepareState()
+                        .execute()
+                        .actionGet()
+                        .getState()
+                        .nodes()
+                        .getClusterManagerNode()
+                        .getName();
+                } catch (Exception e) {
+                    logger.debug("failed to get cluster-manager name", e);
+                    return null;
+                }
+            };
+            assertThat(clusterManagerName.get(), equalTo(nextClusterManagerEligableNodeName));
```

Comment:
So if we swallow the NPE with the try/catch above, won't this test still fail on the assertion here since `clusterManagerName.get()` would be `null` and `nextClusterManagerEligableNodeName` shouldn't be `null`?

Reply:
@jed326 t`assertBusy` allows the assert failed in 10s, so it seems ok.